content/integrations/dyn.md

```diff
@@ -34,9 +33,10 @@ Once you have created a Datadog Read only user:
 4. Configure your [Dyn integration](https://app.datadoghq.com/account/settings#integrations/dyn) inside your Datadog application.
 {{< img src="integrations/dyn/dyn_integration.png" alt="Dyn Integration" style="width:75%;" >}}
 
-5. Select your zones you want to monitor:
-{{< img src="integrations/dyn/dyn_zone.png" alt="Dyn zone" style="width:50%;" >}}
-
+5. Select your zones you want to monitor:<br>
+{{< img src="integrations/dyn/dyn_zone.png" alt="Dyn zone" style="width:75%;" >}}
+The “Zone limitation” applies only to “Zone notes”.
+**Dyn “QPS” metrics are by default collected for all zones**. It's Events and Dyn changes metrics that aren’t.
```

Comment:
Minor edit: `**Dyn “QPS” metrics are by default collected for all zones**, however events generated by Dyn changes are not.`

Are we saying one needs to select the zones to collect events from? What do we mean by Dyn changes?

Wht is the limitation? # of zones to monitor?

Reply:
> It's Events and Dyn changes metrics that aren’t.

Events and Dyn changes are not.